tools/deb/istio-iptables.sh

```diff
@@ -93,6 +93,13 @@ OUTBOUND_IP_RANGES_INCLUDE=${ISTIO_SERVICE_CIDR-}
 OUTBOUND_IP_RANGES_EXCLUDE=${ISTIO_SERVICE_EXCLUDE_CIDR-}
 KUBEVIRT_INTERFACES=
 
+POD_IP=$(hostname --ip-address)
```

Comment:
To be safe, let's remove this for now - I think it is safe if the user sets that explicitly, but attempting to guess is hard to test.

Reply:
If we remove it here then I need a way to flag it somehow to the code.